sc-machine.ini

```diff
@@ -10,7 +10,6 @@ dump_memory_statistics = false
 dump_memory_statistics_period = 1800
 
 storage = ./kb.bin
-extensions = ./bin/extensions
```

Comment:
Now the sc-machine can be started with multiple directories for extensions. I have removed this option from the сonfig because:

- Now debug and release builds of the sc-machine are always in different folders (Debug and Release). In this case there are two ways, either to give two configs (it is not clear why this is necessary and what benefit it gives), or to ask the user to set as an argument
- When starting the sc-machine, you should set not only the directory to the sc-machine extensions, but also the directory to the project extensions. And the logic works that if an option is specified in the config and as an argument, the second one takes precedence. So you can't make it so that you can set one of them

I think we should be moving in a direction where the user can customize for themselves how they will run our entrypoints. He can do it via IDE, scripts, or something else